### DISCUSSION TITLE
Something wrong?

### DISCUSSION THREAD

**Discussion by tbarmo** [cSharpBastard/mmm-nordic-electrical-spot-prices#1] [Q&A]
Hi

After installing this module and trying to configure it, a couple of issues have come forward. The link to Nord Pool have changed. I guess this one is correct now: https://www.nordpoolgroup.com/en/maps/#/nordic

The other issue is that the graph looks a bit squeezed

![image](https://user-images.githubusercontent.com/77986230/208956020-e963b917-0dda-4435-97d0-89f65680b8fa.png)

I am using this in config.js
`{
                        module: 'mmm-nordic-electrical-spot-prices',
                        position: 'top_left',
                        config: {
                                currency: "NOK"
                                }
                },`

What are the correct value for the area-parameter for the Oslo area? From the map, it should be NO1. No matter what I try with area, I get nothing in the squeezed graph.

Could someone have a look at this pleas?

Thanks

**Comment by cSharpBastard** [comment]
Sorry for late answer! I see that it is indeed very squeezed. I don't have anything in the middle on my screen. The table needs more space. It has 24 staples with prices printed right under. 
You should be able to set config: { area: "NO1" } to reflect prices around you. 
Thank you for the update with the url!
If you have any suggestions, pleas feel free to contact me.
